src/components/schemas/CountAgg.yaml

```
@@ -3,8 +3,16 @@ properties:
   count:
     type: integer
     format: int64
+    minimum: 0
     example: 7568
+    description: Nombre d'observations
   value:
     type: integer
     format: int64
     example: 10068
+    description: Nombre de valeurs présentes pour le champ demandé
+  growths:
+    type: array
+    items:
+      $ref: ./Growth.yaml
+    description: Indicateurs de croissance du nombre d'observations
```

:thinking: **thought** : Pour le coup, c'est des indicateurs de croissance de la valeur et pas du nombre d'observations puisque le nombre d'observation c'est le champs `count`. Pour éviter un quiproquo, je te propose ça, même si c'est moins concis, c'est + précis. A adapter si tu trouves mieux mais voilà l'idée : 
```suggestion
    description: Indicateurs de croissance de la valeur, comparée aux valeurs précédentes (uniquement renseignée lors d'une requête d'évolution)
```